src/cljs/vanilla/grid.cljs

```diff
@@ -11,16 +11,21 @@
 (defn fixup-new-widget [widget]
   (merge widget {:data-grid {:x 0 :y 0 :w 5 :h 15}}))
 
+
+;;@TODO - this should move to the vanilla.update-layout ns?
```

Comment:
Agree, this should probably be in the update-layout ns since its legit the name of the event.  Re-comment/delete prn

Reply:
Done! Makes more sense that way